core/schema/container.graphqls

```diff
@@ -157,3 +157,19 @@ input ExecOpts {
     """
     redirectStderr: String
 }
+
+"""
+EnvVariable is a simple key value struct that represent
+an environment variable.
```

Comment:
```suggestion
EnvVariable is a simple key value object that represents an environment variable.
```

* "Object" is the GraphQL term
* I'm not 100% sure if word wrapping does what we want; the other comments have long lines and with single linebreaks for formatting.